[0m12:34:50.186487 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10730f790>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1093a6ec0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1093a6f80>]}


============================== 12:34:50.191711 | 24d5a246-a931-442c-8436-f0abb3e29a1d ==============================
[0m12:34:50.191711 [info ] [MainThread]: Running with dbt=1.7.5
[0m12:34:50.192326 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'profiles_dir': '/Users/nicole/.dbt', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt init', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m12:34:50.192922 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': PosixPath('/Users/nicole/.dbt')}
[0m12:34:50.193299 [info ] [MainThread]: Creating dbt configuration folder at 
[0m12:35:08.377926 [debug] [MainThread]: Starter project path: /Users/nicole/code/de/elt/.venv/lib/python3.10/site-packages/dbt/include/starter_project
[0m12:35:08.384319 [info ] [MainThread]: 
Your new dbt project "custom_postgres" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m12:35:08.384853 [info ] [MainThread]: Setting up your profile.
[0m12:37:17.775119 [error] [MainThread]: Encountered an error:

[0m12:37:17.786326 [error] [MainThread]: Traceback (most recent call last):
  File "/Users/nicole/code/de/elt/.venv/lib/python3.10/site-packages/dbt/cli/requires.py", line 91, in wrapper
    result, success = func(*args, **kwargs)
  File "/Users/nicole/code/de/elt/.venv/lib/python3.10/site-packages/dbt/cli/requires.py", line 76, in wrapper
    return func(*args, **kwargs)
  File "/Users/nicole/code/de/elt/.venv/lib/python3.10/site-packages/dbt/cli/main.py", line 515, in init
    results = task.run()
  File "/Users/nicole/code/de/elt/.venv/lib/python3.10/site-packages/dbt/task/init.py", line 344, in run
    self.setup_profile(profile_name)
  File "/Users/nicole/code/de/elt/.venv/lib/python3.10/site-packages/dbt/task/init.py", line 264, in setup_profile
    self.create_profile_from_target(adapter, profile_name=profile_name)
  File "/Users/nicole/code/de/elt/.venv/lib/python3.10/site-packages/dbt/task/init.py", line 181, in create_profile_from_target
    self.create_profile_from_profile_template(profile_template, profile_name)
  File "/Users/nicole/code/de/elt/.venv/lib/python3.10/site-packages/dbt/task/init.py", line 165, in create_profile_from_profile_template
    target = self.generate_target_from_input(prompts, initial_target)
  File "/Users/nicole/code/de/elt/.venv/lib/python3.10/site-packages/dbt/task/init.py", line 131, in generate_target_from_input
    target[key] = click.prompt(
  File "/Users/nicole/code/de/elt/.venv/lib/python3.10/site-packages/click/termui.py", line 164, in prompt
    value = prompt_func(prompt)
  File "/Users/nicole/code/de/elt/.venv/lib/python3.10/site-packages/click/termui.py", line 147, in prompt_func
    raise Abort() from None
click.exceptions.Abort

[0m12:37:17.810084 [debug] [MainThread]: Resource report: {"command_name": "init", "command_wall_clock_time": 147.64517, "process_user_time": 1.911433, "process_kernel_time": 0.126962, "process_mem_max_rss": "101515264", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m12:37:17.811080 [debug] [MainThread]: Command `dbt init` failed at 12:37:17.810893 after 147.65 seconds
[0m12:37:17.811709 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10730f790>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1094937c0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1094b98d0>]}
[0m12:37:17.812344 [debug] [MainThread]: Flushing usage events
[0m12:37:23.266602 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10896f790>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10becafb0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10becb070>]}


============================== 12:37:23.270929 | 92ab30eb-c2aa-48bb-912d-299b1abb680b ==============================
[0m12:37:23.270929 [info ] [MainThread]: Running with dbt=1.7.5
[0m12:37:23.271679 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'profiles_dir': '/Users/nicole/.dbt', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'log_format': 'default', 'invocation_command': 'dbt init', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m12:37:26.557748 [info ] [MainThread]: A project called custom_postgres already exists here.
[0m12:37:26.560691 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 3.3125648, "process_user_time": 1.891765, "process_kernel_time": 0.128707, "process_mem_max_rss": "100581376", "process_in_blocks": "0", "process_out_blocks": "0"}
[0m12:37:26.562018 [debug] [MainThread]: Command `dbt init` succeeded at 12:37:26.561735 after 3.31 seconds
[0m12:37:26.563131 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10896f790>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10becb190>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10becb130>]}
[0m12:37:26.564169 [debug] [MainThread]: Flushing usage events
[0m12:37:40.036554 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1029ef790>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104a8afe0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104a8b0a0>]}


============================== 12:37:40.042152 | fd1f3dd2-8854-491e-8cee-36fa6e990467 ==============================
[0m12:37:40.042152 [info ] [MainThread]: Running with dbt=1.7.5
[0m12:37:40.042902 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'profiles_dir': '/Users/nicole/.dbt', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt init', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m12:37:43.224813 [debug] [MainThread]: Starter project path: /Users/nicole/code/de/elt/.venv/lib/python3.10/site-packages/dbt/include/starter_project
[0m12:37:43.234730 [info ] [MainThread]: 
Your new dbt project "custom_postgres" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m12:37:43.235305 [info ] [MainThread]: Setting up your profile.
[0m12:38:14.804090 [info ] [MainThread]: Profile custom_postgres written to /Users/nicole/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m12:38:14.807879 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 34.78943, "process_user_time": 1.927456, "process_kernel_time": 0.155818, "process_mem_max_rss": "103284736", "process_in_blocks": "0", "process_out_blocks": "0"}
[0m12:38:14.808678 [debug] [MainThread]: Command `dbt init` succeeded at 12:38:14.808510 after 34.79 seconds
[0m12:38:14.809213 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1029ef790>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104b74df0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104a8b310>]}
[0m12:38:14.809665 [debug] [MainThread]: Flushing usage events
